coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/BusinessLogic/RegistrationBusinessLogic.cs

```diff
@@ -39,17 +40,23 @@ public RegistrationBusinessLogic(IConfiguration configuration, IRegistrationDBAc
 
         public async Task CreateUsersAsync(List<UserCreationInfo> userList, string realm, string token, Dictionary<string, string> userInfo)
         {
-            var manager = new KeycloakIdentityManager(new KeycloakClient(_configuration.GetValue<string>("KeyCloakConnectionString"), () => token), "");
+            var client = new KeycloakClient(_configuration.GetValue<string>("KeyCloakConnectionString"), () => token);
+            var clientId = _configuration.GetValue<string>("KeyCloakClientID");
             foreach (UserCreationInfo user in userList)
             {
-                var newUser = new CreateUser
+                var pwd = new Password();
```

Comment:
I guess creation of new Password instance shall also be done outside the loop (next is supposed to create a new Password on each call)
(sorry I didn't look more closely so this is not the third round ;-) )

Reply:
Again right, thanks :) Done.